src/cmd/sanity_test/sanity_tests.go

```diff
@@ -354,7 +354,7 @@ func checkOnedriveRestoration(
 	getRestoreData(ctx, client, *drive.GetId(), restoreFolderID, restoreFile, restoreFolderPermission, startTime)
 
 	for folderName, permissions := range folderPermission {
-		logger.Ctx(ctx).Info("checking for folder: %s \n", folderName)
+		logger.Ctx(ctx).Info("checking for folder: ", folderName, "\n")
```

Comment:
Should this be having a newline?

Reply:
No. The `logger.Ctx(ctx).Info` function is handling it implicitly.